### PR TITLE
Add mse_v2 tuning strategy

### PR DESCRIPTION
**Pull request by intel-zhangyi**
## Type of Change

New tuning strategy `mse_v2`.

## Description

The implementation of `mse_v2` strategy, supporting PT and TF.

JIRA ticket: 
https://jira.devtools.intel.com/browse/ILITV-2295 (TF Support)

## Expected Behavior & Potential Risk

A new tuning strategy `mse_v2` is imported, which is effective for large models large datasets. 

## How has this PR been tested?

Tested Many models:

* PreCI Test: https://dev.azure.com/lpot-inc/neural-compressor/_build/results?buildId=5250
* Test TF Models: https://inteltf-jenk.sh.intel.com/job/intel-lpot-validation-top-mr-extension/3647/
* Test PT Models: https://inteltf-jenk.sh.intel.com/job/intel-lpot-validation-top-mr-extension/3659/

## Dependency Change?

no dependency changes.

### CODE REVIEW

File: examples/.config/model_params_pytorch.json [hunk 1]
```diff
@@ -18,6 +18,96 @@
       "batch_size": 100,
       "new_benchmark": false
     },
+    "googlenet_fx": {
```
Comment (yiliu30) [inline examples/.config/model_params_pytorch.json:21]: Adding a new strategy should not change the file under `examples`.
Please keep all of the files under `examples` be same as the master.

File: neural_compressor/config.py [hunk 2]
```diff
@@ -521,9 +542,16 @@ def strategy(self):
     @strategy.setter
     def strategy(self, strategy):
         if check_value('strategy', strategy, str,
-            ['basic', 'mse', 'bayesian', 'random', 'exhaustive']):
+            ['basic', 'mse', 'bayesian', 'random', 'exhaustive', 'sigopt', 'tpe']):
```
Comment (yiliu30) [inline neural_compressor/config.py:545]: Please add `mse_v2` to the checklist.

File: neural_compressor/config.py [hunk 3]
```diff
@@ -399,9 +411,17 @@ def strategy(self):
...
```
Comment (yiliu30) [inline neural_compressor/config.py:414]: Please add `mse_v2` to the checklist.

File: test/strategy/test_mse_v2_tensorflow.py [hunk 4]
```diff
@@ -0,0 +1,105 @@
...
+def eval_func(model):
+    #               1, 2, 3, 4, 5, 6, 7, 8, 9, 10
+    eval_list = [0, 1, 0, 0, 0, 1, 1, 0, 0, 1, 1]
+    
+    global i
+    i += 1
+    return eval_list[i]
+
+
+class TestMSEStrategy(unittest.TestCase):
```
Comment (yiliu30) [inline test/strategy/test_mse_v2_tensorflow.py:80]: Please replace the `TestMSEStrategy` with `TestMSEV2Strategy`.

**Review by yiliu30 — APPROVED** [review]

File: examples/.config/model_params_pytorch.json (hunk 1 above)
Comment (intel-zhangyi) [reply examples/.config/model_params_pytorch.json]: resolved: 
909549b1062f69a74b38812f03f025832e20cd21
f47c016308e95fa507539620f4d84c9d189e8e116

File: neural_compressor/config.py (hunk 2 above)
Comment (intel-zhangyi) [reply neural_compressor/config.py]: resolved:
1617594c52897d4f278905978f42e3be23e5e79d

File: neural_compressor/config.py (hunk 3 above)
Comment (intel-zhangyi) [reply neural_compressor/config.py]: resolved:
66a238ba398dede70cb49e4a0d810d6939e54548

File: test/strategy/test_mse_v2_tensorflow.py (hunk 4 above)
Comment (intel-zhangyi) [reply test/strategy/test_mse_v2_tensorflow.py]: resolved:
9360aa34b64e58dee9c984c6986c0d3917231bca

**Comment by ftian1** [comment]
@yiliu30 I didn't see the test report on EfficientNet-Bx series.

**Comment by yiliu30** [comment]
> @yiliu30 I didn't see the test report on EfficientNet-Bx series.

Hi @ftian1, the test result of  EfficientNet-Bx series if here [JOB-3339](https://inteltf-jenk.sh.intel.com/job/intel-lpot-validation-top-mr-extension/3339/artifact/report.html).

**Review by xin3he — APPROVED** [review]

**Comment by ftian1** [comment]
ok. it looks good to me. pls take time to check perf issue when you are free. we can merge this PR now

**Review by ftian1 — APPROVED** [review]

**Comment by intel-zhangyi** [comment]
@chensuyue please merge this PR if there are no more comments.

**Pull request merged**